lib/palindrome_check.rb

```diff
@@ -1,7 +1,25 @@
 # A method to check if the input string is a palindrome.
 # Return true if the string is a palindrome. Return false otherwise.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n), where n is determined by the size of the string "my_phrase"
```

Comment:
Why?

Also, [nit], it's a little clearer to just say "where `n` is the size of the string `my_phrase`".